tests/django/test_models.py

```diff
@@ -250,12 +255,27 @@ def test_model_factory_loose_relations_n_m_temporeel(woningbouwplannen_dataset,
     model_cls = model_dict["woningbouwplan"]
     meta = model_cls._meta
     buurten_field = meta.get_field("buurten")
+    intermediate_table = buurten_field.remote_field.through
     assert isinstance(buurten_field, LooseRelationManyToManyField)
-    assert isinstance(buurten_field.remote_field.through, ModelBase)
+    assert isinstance(intermediate_table, ModelBase)
+
+    assert sorted([x.name for x in intermediate_table._meta.get_fields()]) == [
+        "buurten",
+        "id",
+        "woningbouwplan",
+    ]
```

Comment:
No need for a generator to create a list and sort it afterwards, use sets instead
```python
assert {x.name for x in intermediate_table._meta.get_fields() } == { "buurten", "id", "woningbouwplan"}
```